packages/core/src/render3/util/global_utils.ts

```diff
@@ -71,7 +78,8 @@ const globalUtilsFunctions = {
   'applyChanges': applyChanges,
   'isSignal': isSignal,
 };
-type GlobalUtilsFunctions = keyof typeof globalUtilsFunctions;
+type InternalGlobalUtilsFunctions = keyof typeof globalUtilsFunctions;
```

Comment:
Wdyt of this change ? 
```suggestion
type CoreGlobalUtilsFunctions = keyof typeof globalUtilsFunctions;
```
Internal is kind of vague and every framework module has some sort of internals.

Reply:
Thanks, @JeanMeche, I updated the typings and committed message both.